includes/admin/dashboard-widgets.php

```diff
@@ -54,46 +81,40 @@ function edd_load_dashboard_sales_widget( ) {
 		die();
 	}
 
-	$stats = new EDD_Payment_Stats; ?>
+	$stats = new EDD_Payment_Stats();
+	$data  = edd_get_dashboard_sales_widget_data(); ?>
 	<div class="edd_dashboard_widget">
 		<div class="table table_left table_current_month">
 			<table>
 				<thead>
 					<tr>
-						<td colspan="2"><?php _e( 'Current Month', 'easy-digital-downloads' ) ?></td>
+						<td colspan="2"><?php esc_html_e( 'Current Month', 'easy-digital-downloads' ); ?></td>
 					</tr>
 				</thead>
 				<tbody>
 					<tr>
-						<td class="first t monthly_earnings"><?php _e( 'Earnings', 'easy-digital-downloads' ); ?></td>
-						<td class="b b-earnings"><?php echo edd_currency_filter( edd_format_amount( $stats->get_earnings( 0, 'this_month' ) ) ); ?></td>
+						<td class="first t monthly_earnings"><?php esc_html_e( 'Earnings', 'easy-digital-downloads' ); ?></td>
+						<td class="b b-earnings"><?php echo esc_attr( $data['this_month']['earnings'] ); ?></td>
```

Comment:
These should all use `esc_html()` if they are inside of HTML tags versus HTML attributes.

```suggestion
						<td class="b b-earnings"><?php echo esc_html( $data['this_month']['earnings'] ); ?></td>
```